proto/qdrant/cloud/booking/v1/booking.proto

```diff
@@ -114,6 +114,16 @@ enum PackageStatus {
   PACKAGE_STATUS_DEACTIVATED = 2;
 }
 
+// PackageStatus defines the valid states a package can be in.
+enum PackageTier {
+  // Unspecified pricing tier, necessary default.
```

Comment:
pricing tier --> package tier ?

Reply:
please drop: `, necessary default.` we don't do this on others, prefer to be consistent